src/formatDate.js

```diff
@@ -50,7 +50,36 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const initialDateFromat = date.split(fromFormat[3]);
+  const result = [];
+
+  for (let i = 0; i < toFormat.length - 1; i++) {
+    for (let j = 0; j < fromFormat.length - 1; j++) {
+      if (fromFormat[j] === toFormat[i]) {
+        result.push(initialDateFromat[j]);
+      }
+
+      if (fromFormat[j] === 'YYYY' && toFormat[i] === 'YY') {
```

Comment:
Create a variables for year format, it was repeated a couple of times

Reply:
done